icu4c/source/samples/datefmt/README.TXT

```diff
@@ -1,107 +0,0 @@
-Copyright (C) 2016 and later: Unicode, Inc. and others.
-License & terms of use: http://www.unicode.org/copyright.html
-
-Copyright (c) 2002-2010, International Business Machines Corporation and others. All Rights Reserved.
-
-IMPORTANT:
-
-This sample was originally intended as an exercise for the ICU Workshop (September 2000).
-The code currently provided in the solution file is the answer to the exercises, each step can still be found in the 'answers' subdirectory.
-
-
-
-** Workshop homepage is:  
-  http://www.icu-project.org/docs/workshop_2000/agenda.html
-
-  #Date/Time/Number Formatting Support
-  9:30am - 10:30am
-  Alan Liu
-
-  Topics:
-  1. What is the date/time support in ICU?
-  2. What is the timezone support in ICU?
-  3. What kind of formatting and parsing support is available in ICU, i.e.
-  NumberFormat, DateFormat, MessageFormat?
```

Comment:
@srl295 this also looks elaborate and interesting. Worth something?

Reply:
Let's please keep this one for now.

Reply:
makes sense to me, as long as it builds